Source/JavaScriptCore/runtime/MachineContext.h

```diff
@@ -242,14 +224,8 @@ static inline void*& framePointerImpl(PlatformRegisters& regs)
 
 #if __DARWIN_UNIX03
 
-#if CPU(X86)
-    return reinterpret_cast<void*&>(regs.__ebp);
-#elif CPU(X86_64)
+#if CPU(X86_64)
     return reinterpret_cast<void*&>(regs.__rbp);
-#elif CPU(ARM_THUMB2)
-    return reinterpret_cast<void*&>(regs.__r[7]);
-#elif CPU(ARM)
-    return reinterpret_cast<void*&>(regs.__r[11]);
```

Comment:
I think you didn't want to delete this.

Reply:
It is actually fine since we do not need to have MachineContext for that (Darwin 32bit ARM). It is optional and not necessary unless we are using JIT.